testnet/stacks-node/src/tests/signer/commands/stalling.rs

```diff
@@ -0,0 +1,60 @@
+use super::context::{SignerTestContext, SignerTestState};
+use madhouse::{Command, CommandWrapper};
+use proptest::prelude::{Just, Strategy};
+use std::sync::Arc;
+
+pub struct StallMiningCommand;
+
+impl Command<SignerTestState, SignerTestContext> for StallMiningCommand {
+    fn check(&self, state: &SignerTestState) -> bool {
+        println!(
+            "Checking: Stalling mining. Result: {:?}",
+            !state.mining_stalled
+        );
```

Comment:
Don't use `println!` - use `info!` or `test_debug!` please

Reply:
Addressed in https://github.com/stacks-network/stacks-core/pull/6007/commits/604680729af36d3f7b39b5eea0ddec72c4cd52ef.